include/WCSimWCDigi.hh

```diff
@@ -195,43 +199,48 @@ public:
 
     for (i = 1; i < (int) time.size(); ++i)
       {
-        index_time  = time.at(i);
-        index_timepresmear  = time_presmear.at(i);
-        index_pe = pe.at(i);
-        if(sort_digi_compositions) index_digicomp = fDigiComp.at(i);
-        index_trackID = trackID.at(i);
-        index_parentSavedTrackID = parentSavedTrackID.at(i);
-	index_photonstarttime = photonStartTime[i];
-	index_photonstartpos = photonStartPos[i];
-	index_photonendpos = photonEndPos[i];
-	index_photonstartdir = photonStartDir[i];
-	index_photonenddir = photonEndDir[i];
-        for (j = i; j > 0 && time.at(j-1) > index_time; j--) {
-          time.at(j) = time.at(j-1);
-          time_presmear.at(j) = time_presmear.at(j-1);
-          pe.at(j) = pe.at(j-1);
-          if(sort_digi_compositions) fDigiComp.at(j) = fDigiComp.at(j-1);
-          trackID.at(j) = trackID.at(j-1);
-          parentSavedTrackID.at(j) = parentSavedTrackID.at(j-1);
-	  photonStartTime.at(j) = photonStartTime.at(j-1);
-	  photonStartPos.at(j) = photonStartPos.at(j-1);
-	  photonEndPos.at(j) = photonEndPos.at(j-1);
-	  photonStartDir.at(j) = photonStartDir.at(j-1);
-	  photonEndDir.at(j) = photonEndDir.at(j-1);
-          //G4cout <<"swapping "<<time.at(j-1)<<" "<<index_time<<G4endl;
+        index_time         = time.at(i);
+        index_timepresmear = time_presmear.at(i);
+        index_pe           = pe.at(i);
+        if(sort_digi_compositions) 
+          index_digicomp             = fDigiComp.at(i);
+          index_trackID              = trackID.at(i);
+          index_parentSavedTrackID   = parentSavedTrackID.at(i);
+          index_photonstarttime      = photonStartTime[i];
+          index_photonstartpos       = photonStartPos[i];
+          index_photonendpos         = photonEndPos[i];
+          index_photonstartdir       = photonStartDir[i];
+          index_photonenddir         = photonEndDir[i];
+          index_photoncreatorprocess = photonCreatorProcess[i];
+
+          for (j = i; j > 0 && time.at(j-1) > index_time; j--) {
+            time.at(j) = time.at(j-1);
+            time_presmear.at(j) = time_presmear.at(j-1);
+            pe.at(j) = pe.at(j-1);
+            if(sort_digi_compositions) fDigiComp.at(j) = fDigiComp.at(j-1);
+              trackID.at(j) = trackID.at(j-1);
```

Comment:
don't have the extra indentation with respect to the `if` above since they are not connected

Reply:
👍